ui/redux/reducers/comments.js

```diff
@@ -299,9 +300,16 @@ export default handleActions(
       const isLoadingByParentId = Object.assign({}, state.isLoadingByParentId);
       const settingsByChannelId = Object.assign({}, state.settingsByChannelId);
 
+      // save an array of claim ids of members-only chats to check during list
+      let membersOnlyChats = settingsByChannelId[creatorClaimId].members_only_chat || [];
+      if (restrictedToMembersOnly) {
+        membersOnlyChats.push(claimId);
+      }
+
```

Comment:
:small_red_triangle: This might be the wrong structure to place it in.  `settingsByChannelId` reflects settings for the entire channel, but `members_only_chat` can be on a per-claim basis, right?

Also, is `members_only_chat` now part of the `settings.Get` return structure?  The structure is also used for the "set" part ... not sure if it's right to combine here (although it does seem convenient compared to creating another storage).  That would explain the lint error.  Maybe ok for now...

Reply:
:small_red_triangle: http://localhost:9090/@timeout:8/HelloDrone:5

It could be a misconfiguration, but I don't think this is gated content.

<img width="244" alt="image" src="https://user-images.githubusercontent.com/64950861/192417748-14ffc008-8e8a-4ce5-88ce-0359530a032e.png">